src/accounts/views.py

```diff
@@ -151,3 +152,47 @@ def get_success_url(self):
 
     def get_object(self):
         return self.request.user
+
+
+class NewsletterView(FormView):
+    """Allow users to subscribe newsletter."""
+
+    template_name = 'accounts/newsletter.html'
+    form_class = NewsletterForm
+    success_url = reverse_lazy('newsletter_success')
+
+    def get_newsletter_user(self):
+        if self.request.method == 'POST':
+            form = NewsletterForm(self.request.POST)
+            if form.is_valid():
+                self.export_account(form.cleaned_data['email'])
+                return HttpResponseRedirect('/newsletter_success/')
+        else:
+            form = NewsletterForm()
+
+    def export_account(self, user):
```

Comment:
In fine il faudrait qu'on ait 1 seule méthode `export_account()` au lieu de la dupliquer. Du coup modifier la management command `export_account.py` pour faire appel à cette méthode directement ?